pcsx2-qt/MainWindow.cpp

```diff
@@ -1748,6 +1748,12 @@ void MainWindow::onInputRecNewActionTriggered()
 						m_ui.actionToolbarReset->setEnabled(!g_InputRecording.isTypeSavestate());
 					});
 				}
+				else
```

Comment:
Ugh, this whole function is pretty gross.

But, just use Host:: ReportErrorAsync rather than explicitly queuing if it's running on the CPU thread. 

(Can use the TRANSLATE macro for l10n)

Reply:
I missed the existence of the TRANSLATE macro

Amended